hs/t/y/many_itxns.txt

```diff
@@ -4,4 +4,5 @@ Verifying for generic connector
   Verifying when NO participants are honest
 Checked 517 theorems; No failures!
 WARNING: Compiler instructed to emit for Algorand, but we can statically determine that this program will not work on Algorand, because:
- * Step 0 uses 261 inner transactions, but the limit is 256. Step 0 starts at ./many_itxns.rsh:13:5:dot.
+ * Step 0 uses 264 inner transactions, but the limit is 256. Step 0 starts at ./many_itxns.rsh:13:5:dot.
+ * Too many scratch slots
```

Comment:
Why do you think this program changed?

Reply:
The program makes 1 pay and 256 transfers. So there are 256 assumptions, that balance doesn't overflow/wraparound, added via `doBalanceUpdate`

Reply:
I get why there are too many scratch slots. I mean, I don't know where the 3 extra inner txns came from. Maybe from cost?

Reply:
I think so, I don't quite understand the TEAL code inside of the publish handler, but it seems like it's making three more itxns to the companion contract